cvat-ui/src/components/cloud-storages-page/top-bar.tsx

```diff
@@ -12,6 +12,8 @@ import { CloudStoragesQuery } from 'reducers';
 import Input from 'antd/lib/input';
 import { SortingComponent, ResourceFilterHOC, defaultVisibility } from 'components/resource-sorting-filtering';
 
+import dimensions from 'components/projects-page/dimensions';
```

Comment:
Should I create a seperate component which is a Col component and has these demensions because these have been imported from projects-page folder into 6 files.